internal/pkg/utils/orderedmap/orderedmap.go

```diff
@@ -143,20 +143,20 @@ func decodeOrderedMap(dec *json.Decoder, o *OrderedMap) error {
 			switch delim {
 			case '{':
 				if values, ok := o.values[key].(map[string]interface{}); ok {
-					newMap := OrderedMap{
+					newMap := &OrderedMap{
```

Comment:
Vyrieseny hlavny problem s `orderedmap` ... a to ze root typ bol `*OrderedMap`, ale pri JSON decode boli vnorene hodnoty nie-pointer `OrderedMap`.

A preto nebolo mozne pridat nejaku nested hodnotu, bez toho, aby sa nova map ulozila aj do vsetkych parents.
Kedze interna slice `keys` sa zmenila iba v kopii `OrderedMap`. `Set` fungoval aj v starom kode spravne, ... kedze sa slice `keys` neprepisuje iba meni (pocet prvkov zostava rovnaky).

Reply:
Prvy commit je zmena v libke.